GardenApp/app/src/main/AndroidManifest.xml

```diff
@@ -6,15 +6,14 @@
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
 
     <application
+        android:name=".App"
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
-        android:theme="@style/AppTheme"
-        android:name="com.cs1530_group1.gardenapp.App">
```

Comment:
This name was changed to ".App" with the update. This may have been automatic, but the previous name was far more descriptive. We might want to change it back.